src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/ov_plugin/version.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_plugin/version.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         VersionTests,
+                         ::testing::Values(ov::test::utils::DEVICE_GPU),
+                         VersionTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Hetero_BehaviorTests,
+                         VersionTests,
+                         ::testing::Values(ov::test::utils::DEVICE_HETERO),
```

Comment:
As my understanding, we should remove meta plugin tests cases from hardware plugin and put them 
 into conformance tests, am i right?

Reply:
For the moment it isn't clear for me how tests should migrate to conformance scope. So let's merge as is in order to get rid of API1.0 cases, and when only 2.0 cases will be available, it should be easier to migrate

@iefode @sbalandi FYI

Reply:
Hi, @riverlijunjie @vurusovs meta plugins in the conformance scope are tested with the TEMPLATE plugin now.
All API 1.0 tests have been removed from the conformance scope, but there are some tests that have not yet been moved to the 2.0 scope and version tests were among them, so I will add these tests to conformance, thank you  !